src/features/side_bar/components/SideBar/ToggleButton.tsx

```
@@ -34,26 +34,27 @@ export function ToggleButton({ isOpen, setIsOpen }: ToggleButtonProps) {
+          variant="invert"
         >
           {i18n.t('sidebar.collapse')}
-        </ActionsBarBTN>
+        </Button>
       </div>
     );
 
   return (
-    <Tooltip placement="right" offset={6}>
+    <SimpleTooltip content={i18n.t('sidebar.expand')} placement="right" offset={6}>
       <div onClick={toggleIsOpen}>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)